### ISSUE TITLE
Permission tests crash on implementing site due to example_project_app reference

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-core#1211]
When integrating current v0.13.0 dev with SODAR, I'm experiencing a large number of permission test crashes resulting from a `example_project_app` refefence.

`example_project_app` is not included in the installed `django-sodar-core` package and should not be hardcoded anywhere.

Most likely this is due to some app settings test setup. Will look into it. Posting a full dump of one of the crashing test cases in comments.

The crash happens (at least) when making a request using `TestPermissionMixin.send_request()`.

**Comment by mikkonie** [comment]
Example of error dump:
```
======================================================================
ERROR: test_version_update (samplesheets.tests.test_permissions.TestSampleSheetsPermissions)
Test SheetVersionUpdateView permissions
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/urls/base.py", line 71, in reverse
    extra, resolver = resolver.namespace_dict[ns]
KeyError: 'example_project_app'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mikkopen/code/sodar/samplesheets/tests/test_permissions.py", line 858, in test_version_update
    self.assert_response(url, good_users, 200)
  File "/home/mikkopen/.virtualenvs/sodar14/src/django-sodar-core/projectroles/tests/test_permissions.py", line 93, in assert_response
    response = self.send_request(url, method, req_kwargs)
  File "/home/mikkopen/.virtualenvs/sodar14/src/django-sodar-core/projectroles/tests/test_permissions.py", line 47, in send_request
    return req_method(url, **req_kwargs)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/client.py", line 742, in get
    response = super().get(path, data=data, secure=secure, **extra)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/client.py", line 396, in get
    return self.generic('GET', path, secure=secure, **{
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/client.py", line 473, in generic
    return self.request(**r)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/client.py", line 719, in request
    self.check_exception(response)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/client.py", line 580, in check_exception
    raise exc_value
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/core/handlers/base.py", line 204, in _get_response
    response = response.render()
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/response.py", line 83, in rendered_content
    return template.render(context, self._request)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 170, in render
    return self._render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/loader_tags.py", line 195, in render
    return template.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 172, in render
    return self._render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/test/utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/template/defaulttags.py", line 449, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/home/mikkopen/.virtualenvs/sodar14/lib/python3.8/site-packages/django/urls/base.py", line 82, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'example_project_app' is not a registered namespace
```

**Comment by mikkonie** [comment]
References to `example_project_app` in code (not counting the app itself):

```
userprofile/tests/test_views.py:117:            'settings.example_project_app.user_str_setting'
userprofile/tests/test_views.py:122:            'settings.example_project_app.user_int_setting'
userprofile/tests/test_views.py:128:                'settings.example_project_app.user_str_setting_options'
userprofile/tests/test_views.py:133:                'settings.example_project_app.user_int_setting_options'
userprofile/tests/test_views.py:138:                'settings.example_project_app.user_bool_setting'
userprofile/tests/test_views.py:143:                'settings.example_project_app.user_json_setting'
userprofile/tests/test_views.py:187:            'settings.example_project_app.user_str_setting': 'another-text',
userprofile/tests/test_views.py:188:            'settings.example_project_app.user_int_setting': '123',
userprofile/tests/test_views.py:189:            'settings.example_project_app.user_str_setting_options': 'string2',
userprofile/tests/test_views.py:190:            'settings.example_project_app.user_int_setting_options': 1,
userprofile/tests/test_views.py:191:            'settings.example_project_app.user_bool_setting': False,
userprofile/tests/test_views.py:192:            'settings.example_project_app.'
userprofile/tests/test_views.py:194:            'settings.example_project_app.user_callable_setting': 'Test',
userprofile/tests/test_views.py:195:            'settings.example_project_app.user_callable_setting_options': str(
userprofile/tests/test_ui.py:47:                'div[id="div_id_settings.example_project_app.'
userprofile/tests/test_ui.py:53:            'div[id="div_id_settings.example_project_app.'
config/urls.py:54:    path('examples/project/', include('example_project_app.urls')),
config/settings/base.py:96:    'example_project_app.apps.ExampleProjectAppConfig',
timeline/tests/test_templatetags.py:201:    def test_get_app_icon_html_example_project_app(self):
timeline/tests/test_templatetags.py:203:        plugin = get_app_plugin('example_project_app')
timeline/tests/test_templatetags.py:210:            app='example_project_app',
projectroles/tests/test_permissions_api.py:1031:            'app_name': 'example_project_app',
projectroles/tests/test_permissions_api.py:1083:            'app_name': 'example_project_app',
projectroles/tests/test_permissions_api.py:1137:            'app_name': 'example_project_app',
projectroles/tests/test_permissions_api.py:1155:            'app_name': 'example_project_app',
projectroles/tests/test_permissions_api.py:1218:            'app_name': 'example_project_app',
projectroles/tests/test_permissions_api.py:1280:            'app_name': 'example_project_app',
projectroles/tests/test_permissions_api.py:1297:            'app_name': 'example_project_app',
projectroles/tests/test_permissions_api.py:1339:            'app_name': 'example_project_app',
projectroles/tests/test_permissions_api.py:1353:            'app_name': 'example_project_app',
projectroles/tests/test_permissions_api.py:1400:            'app_name': 'example_project_app',
projectroles/tests/test_commands.py:48:EXAMPLE_APP_NAME = 'example_project_app'
projectroles/tests/test_commands.py:516:        self.plugin = Plugin.objects.get(name='example_project_app')
projectroles/tests/test_commands.py:608:                            'settings.example_project_app.ghost',
projectroles/tests/test_commands.py:645:                            'settings.example_project_app.'
projectroles/tests/test_commands.py:690:                            'settings.example_project_app.'
projectroles/tests/test_views.py:106:EXAMPLE_APP_NAME = 'example_project_app'
projectroles/tests/test_views.py:978:        ps['settings.example_project_app.project_int_setting'] = 1
projectroles/tests/test_views.py:979:        ps['settings.example_project_app.project_str_setting'] = 'test'
projectroles/tests/test_views.py:980:        ps['settings.example_project_app.project_bool_setting'] = True
projectroles/tests/test_views.py:981:        ps['settings.example_project_app.project_json_setting'] = '{}'
projectroles/tests/test_views.py:983:            'settings.example_project_app.project_callable_setting'
projectroles/tests/test_views.py:986:            'settings.example_project_app.project_callable_setting_options'
projectroles/tests/test_views.py:1261:                'settings.example_project_app.project_str_setting'
projectroles/tests/test_views.py:1267:                'settings.example_project_app.project_int_setting'
projectroles/tests/test_views.py:1273:                'settings.example_project_app.project_bool_setting'
projectroles/tests/test_views.py:1279:                'settings.example_project_app.project_callable_setting'
projectroles/tests/test_views.py:1285:                'settings.example_project_app.project_callable_setting_options'
projectroles/tests/test_views.py:1303:        values['settings.example_project_app.project_int_setting'] = 0
projectroles/tests/test_views.py:1304:        values['settings.example_project_app.project_int_setting_options'] = 0
projectroles/tests/test_views.py:1305:        values['settings.example_project_app.project_str_setting'] = 'test'
projectroles/tests/test_views.py:1307:            'settings.example_project_app.project_str_setting_options'
projectroles/tests/test_views.py:1309:        values['settings.example_project_app.project_bool_setting'] = True
projectroles/tests/test_views.py:1311:            'settings.example_project_app.project_callable_setting'
projectroles/tests/test_views.py:1314:            'settings.example_project_app.project_callable_setting_options'
projectroles/tests/test_views.py:1626:            'settings.example_project_app.project_str_setting'
projectroles/tests/test_views.py:1631:            'settings.example_project_app.project_int_setting'
projectroles/tests/test_views.py:1637:                'settings.example_project_app.project_str_setting_options'
projectroles/tests/test_views.py:1642:                'settings.example_project_app.project_int_setting_options'
projectroles/tests/test_views.py:1647:                'settings.example_project_app.project_bool_setting'
projectroles/tests/test_views.py:1652:                'settings.example_project_app.project_json_setting'
projectroles/tests/test_views.py:1722:            'settings.example_project_app.project_str_setting': 'updated',
projectroles/tests/test_views.py:1723:            'settings.example_project_app.project_int_setting': 170,
projectroles/tests/test_views.py:1724:            'settings.example_project_app.'
projectroles/tests/test_views.py:1726:            'settings.example_project_app.project_int_setting_options': 1,
projectroles/tests/test_views.py:1727:            'settings.example_project_app.project_bool_setting': True,
projectroles/tests/test_views.py:1728:            'settings.example_project_app.'
projectroles/tests/test_views.py:1730:            'settings.example_project_app.'
projectroles/tests/test_views.py:1917:                'settings.example_project_app.project_str_setting'
projectroles/tests/test_views.py:1922:                'settings.example_project_app.project_int_setting'
projectroles/tests/test_views.py:1927:                'settings.example_project_app.project_str_setting_options'
projectroles/tests/test_views.py:1932:                'settings.example_project_app.project_int_setting_options'
projectroles/tests/test_views.py:1937:                'settings.example_project_app.project_bool_setting'
projectroles/tests/test_views.py:1942:                'settings.example_project_app.project_json_setting'
projectroles/tests/test_views.py:1947:                'settings.example_project_app.project_callable_setting'
projectroles/tests/test_views.py:1952:                'settings.example_project_app.project_callable_setting_options'
projectroles/tests/test_views.py:2020:            'settings.example_project_app.project_str_setting': 'updated',
projectroles/tests/test_views.py:2021:            'settings.example_project_app.project_int_setting': 170,
projectroles/tests/test_views.py:2022:            'settings.example_project_app.'
projectroles/tests/test_views.py:2024:            'settings.example_project_app.project_int_setting_options': 1,
projectroles/tests/test_views.py:2025:            'settings.example_project_app.project_bool_setting': True,
projectroles/tests/test_views.py:2026:            'settings.example_project_app.'
projectroles/tests/test_views.py:2028:            'settings.example_project_app.'
projectroles/tests/test_views.py:2030:            'settings.example_project_app.'
projectroles/tests/test_views.py:2221:                'settings.example_project_app.project_str_setting'
projectroles/tests/test_views.py:2226:                'settings.example_project_app.project_int_setting'
projectroles/tests/test_views.py:2231:                'settings.example_project_app.project_str_setting_options'
projectroles/tests/test_views.py:2236:                'settings.example_project_app.project_int_setting_options'
projectroles/tests/test_views.py:2241:                'settings.example_project_app.project_bool_setting'
projectroles/tests/test_views.py:2246:                'settings.example_project_app.project_json_setting'
projectroles/tests/test_views.py:2320:            'settings.example_project_app.project_str_setting': 'updated',
projectroles/tests/test_views.py:2321:            'settings.example_project_app.project_int_setting': 170,
projectroles/tests/test_views.py:2322:            'settings.example_project_app.'
projectroles/tests/test_views.py:2324:            'settings.example_project_app.project_int_setting_options': 1,
projectroles/tests/test_views.py:2325:            'settings.example_project_app.project_bool_setting': True,
projectroles/tests/test_views.py:2326:            'settings.example_project_app.'
projectroles/tests/test_views.py:2328:            'settings.example_project_app.'
projectroles/tests/test_views.py:2330:            'settings.example_project_app.'
projectroles/tests/test_templatetags.py:154:                'example_project_app',
projectroles/tests/test_templatetags.py:162:                'example_project_app', 'user_bool_setting', user=self.user
projectroles/tests/test_models.py:43:EXAMPLE_APP_NAME = 'example_project_app'
projectroles/tests/test_ui.py:1613:            By.ID, 'div_id_settings.example_project_app.project_int_setting'
projectroles/tests/test_ui.py:1626:            By.ID, 'div_id_settings.example_project_app.project_int_setting'
projectroles/tests/test_ui.py:1639:            By.ID, 'div_id_settings.example_project_app.project_int_setting'
projectroles/tests/test_ui.py:1652:            By.ID, 'div_id_settings.example_project_app.project_int_setting'
projectroles/tests/test_ui.py:1665:            By.ID, 'div_id_settings.example_project_app.project_int_setting'
projectroles/tests/test_ui.py:1687:            'div[id="div_id_settings.example_project_app.project_int_setting"]',
projectroles/tests/test_ui.py:1727:            By.ID, 'div_id_settings.example_project_app.project_int_setting'
projectroles/tests/test_ui.py:1740:            By.ID, 'div_id_settings.example_project_app.project_int_setting'
projectroles/tests/test_app_settings.py:15:from example_project_app.plugins import (
projectroles/tests/test_app_settings.py:40:EXAMPLE_APP_NAME = 'example_project_app'
projectroles/tests/test_app_settings.py:49:        # TODO: Rename these to match the settings in example_project_app
projectroles/tests/test_views_api.py:84:EX_APP_NAME = 'example_project_app'
```

**Comment by mikkonie** [comment]
Fixed by #1212.
